README.rst

```diff
@@ -1,53 +1,104 @@
 bobtemplates.itpp.odoo
-=================
+======================
 
 ``bobtemplates.itpp.odoo`` is a set of `mr.bob
 <https://mrbob.readthedocs.io/en/latest/>`_
 templates to use when developing Odoo addons.
 
-It provides the following templates:
+* It provides the following templates:
+
+  * ``addon`` : Directory with minimum files, that allows to install addon in odoo
+  * ``controller`` : Controller that prints "Hello World" by the url ``{ODOO_ROOT_URL}/hello/world``
+  * ``css`` : Empty ``.css``
+  * ``data`` : Empty ``.xml``
+  * ``demo`` : Empty ``.xml``
+  * ``js`` : ``.js``-file and ``.xml``-file for applying JS
+  * ``qweb`` : Empty ``.xml``-file in ``{ADDON_DIRECTORY}/static/src/xml``-directory
+  * ``test`` : ``.py``-file in ``tests`` directory
+  * ``view`` : Empty ``.xml``-file in ``views``-directory
+  * ``wizard`` : 
+  
+Install
+~~~~~~~
+* Install via pip from github-repository:
 
-  * ``addon``: an addon skeletton, with optional OCA README and icon
+  .. code:: shell
 
-In future:
+    pip install git+https://github.com/it-projects-llc/bobtemplates.odoo.git@master
 
-  * ``model``: an Odoo model with accompanying form, tree, action, menu,
-    demo data and ACL
-  * ``test``: a test class
-  * ``wizard``: a wizard with transient model, view and action
+Configuration
+~~~~~~~~~~~~~
 
-Install
-~~~~~~~
+* Set copyright variables in ``.mrbob.ini`` in ``/path/to/config``:
 
   .. code:: shell
 
-    pip install git+https://github.com/it-projects-llc/bobtemplates.odoo.git@master
+    mrbob bobtemplates.itpp:copyright --remember-answers -O /path/to/config
 
 Usage
-~~~~~~~~~~~~~~~~~~~~~~~
-
-* To create addon-template in current directory (interactive):
+~~~~~
+    
+* To create template in the current directory (interactive):
 
   .. code:: shell
+    
+    mrbob --config ```/path/to/config/.mrbob.ini``` bobtemplates.itpp.odoo:{NAME_OF_TEMPLATE}
+    
+    ### where {NAME_OF_TEMPLATE} is:
+    ###
+    ### For creating something else than addon-template:
+    ### --- add "-O /path/to/addon/directory" at the end of command ---
+
+    * addon
+    * controller
+    * css
+    * data
+    * demo
+    * js
+    * model
+    * qweb
+    * test
+    * view
+    * wizard
+    
+    ### Example: 
+    ### Creation of addon-template
+
+    mrbob --config /path/to/config/.mrbob.ini bobtemplates.itpp.odoo:addon
+
+
+* To create template in ``/path/to/directory``:
 
-    mrbob bobtemplates.itpp.odoo:addon
+  .. code:: shell
 
-* To create addon-template in current directory (non-interactive):
+    mrbob ... -O /path/to/directory
 
+* To set default values of variables put code in ``/path/to/config/.mrbob.ini``:
+  
   .. code:: shell
 
-    mrbob bobtemplates.itpp.odoo:addon -n
+      [variables]
```

Comment:
there was a notice about it somewhere above, and here it is again. But still, there is no proper example of available variables. Not sure that putting a version here is a good idea